source/_integrations/nest.markdown

```diff
@@ -41,6 +41,12 @@ The Nest Smart Device Management (SDM) API **requires a US$5 fee**.
 
 </div>
 
+<div class='note'>
+
+**Home Assistant {{ site.current_major_version }}.{{ site.current_minor_version }}**: The Home Assistant documentation website always reflects the current version of Home Assistant.
```

Comment:
This is not fitting for an integration page, as it applies to every page in our documentation.